django/db/models/fields/related.py

```diff
@@ -1145,6 +1151,8 @@ def get_db_prep_value(self, value, connection, prepared=False):
         return self.target_field.get_db_prep_value(value, connection, prepared)
 
     def get_prep_value(self, value):
+        if 1 < len(self.from_fields):
+            return super().get_prep_value(value)
```

Comment:
It seems like we should still be delegating to the `CompositePrimaryKey.get_prep_value`?

Reply:
There's no `CompositePrimaryKey` in this PR, it's based on the `main` branch.